diagnostic.md

```diff
@@ -150,7 +172,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will "float" to the right side of the screen. divTwo, Three and Five will
+float to the left of the screen and divFour should clear the left float and should be
+positioned below the left floats.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
2 3      1
4 5